appinfo/info.xml

```diff
@@ -34,7 +34,8 @@ When using OAuth2 a unique access token is generated for each device or third pa
     <screenshot>https://raw.githubusercontent.com/owncloud/screenshots/master/oauth2/ownCloud-oauth2-app-auth-request.jpg</screenshot>
     <screenshot>https://raw.githubusercontent.com/owncloud/screenshots/master/oauth2/ownCloud-oauth2-app-authorized.jpg</screenshot>
     <dependencies>
-        <owncloud min-version="10.3" max-version="10"/>
+        <owncloud min-version="10.11" max-version="10" />
```

Comment:
We should not do this in this release.

The included fix shall be made available to as many oc installations (versions) as possible